pandas/core/groupby/ops.py

```diff
@@ -497,6 +509,8 @@ def _call_cython_op(
         values = values.T
         if mask is not None:
             mask = mask.T
+            if mask_out is not None:
```

Comment:
should be out-dented

Reply:
no, mask_out is not None in a strict subset of cases when mask is not None

Reply:
sure but it has the guard on it so shouldn't make any difference and it IS indepenent of mask no?

Reply:
> so shouldn't make any difference

tiny difference in an unnecessary check being avoided.  mostly clarity

> and it IS indepenent of mask no?

no